recipes/test.rb

```diff
@@ -28,7 +28,19 @@
 include_recipe 'jenkins::master'
 
 jenkins_plugins_workflow 'workflow-job' do
-  script 'hello-world.groovy'
+  script 'workflow/hello-world.groovy'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.